integreat_cms/cms/views/imprint/imprint_form_view.py

```diff
@@ -92,25 +93,34 @@ def get(self, request, *args, **kwargs):
 
         disabled = False
         if imprint:
-            # Show information if latest changes are only saved as draft
+            # Show information if latest changes are only saved as draft, but there is an earlier public version of this translation
             public_translation = imprint.get_public_translation(language.slug)
-            if public_translation and imprint_translation != public_translation:
+            if (
+                public_translation
+                and imprint_translation != public_translation
+                and public_translation.id
+            ):
+                public_translation_url = reverse(
+                    "imprint_revisions",
+                    kwargs={
+                        "region_slug": region.slug,
+                        "language_slug": language.slug,
+                        "selected_revision": public_translation.version,
+                    },
+                )
+
                 messages.info(
                     request,
-                    _(
-                        "This is <b>not</b> the most recent public revision of this translation. Instead, <a href='%(revision_url)s' class='underline hover:no-underline'>revision %(revision)s</a> is shown in the apps."
-                    )
-                    % {
-                        "revision_url": reverse(
-                            "imprint_revisions",
-                            kwargs={
-                                "region_slug": region.slug,
-                                "language_slug": language.slug,
-                                "selected_revision": public_translation.version,
-                            },
+                    format_html(
+                        "{} <a href='{}' class='underline hover:no-underline'>{} {}</a> {}",
+                        _(
+                            "This is not the most recent public revision of this translation. Instead,"
```

Comment:
```suggestion
                    translate_link(
                        message,
                        attributes={
                            "href": public_translation_url,
                            "class": "underline hover:no-underline",
                        },
```
The problem with splitting the sentence is that the order of the words might be different in other languages and it makes it harder to grasp the context of the translations.
Instead, I think this would be a good use case for my [translate_link()](https://github.com/digitalfabrik/integreat-cms/blob/45a836a6fb8426d0183a8c760b1895dba930c733/integreat_cms/cms/utils/translation_utils.py#L26-L48) utility I recently implemented.

Reply:
Yep, looks better indeed. Thanks